File: MyLanguageWeb/src/main/resources/static/js/classes/dto/other/response/value/date_response_dto.js
export class DateResponseDTO {
    #value;

    constructor(json) {
        if (json) {
            let value = json["value"];
            if (value) {
                this.#value = new Date(value);
            }
        }
    }

    getValue() {
        return this.#value;
    }
}

File: MyLanguageWeb/src/main/java/ru/herooo/mylanguageweb/dto/other/response/value/DateResponseDTO.java
package ru.herooo.mylanguageweb.dto.other.response.value;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.time.LocalDate;

public class DateResponseDTO {
    @JsonProperty("value")
    private LocalDate value;

    public DateResponseDTO(LocalDate value) {
        this.value = value;
    }

    public LocalDate getValue() {
        return value;
    }

    public void setValue(LocalDate value) {
        this.value = value;
    }
}
